YSE_App/templates/YSE_App/dashboard.html

```diff
@@ -7,7 +7,7 @@
 
 {% block content_header %}
     <h1>
-        CHIME/FRB Outriggers
+        YSE/PZ
```

Comment:
Why YSE/PZ and not CHIME/FRB Outriggers?

Reply:
we use `frb_dashboard.html`

this is the original (and I reverted it back)